test/modules/TestResultCoordinator/TestOperationResultStorage.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.Storage;
+    using Microsoft.Azure.Devices.Edge.Storage.RocksDb;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    class TestOperationResultStorage
+    {
+        static readonly ILogger Logger = ModuleUtil.CreateLogger("TestResultCoordinator");
+        Dictionary<string, ISequentialStore<TestOperationResult>> resultStores;
+
+        public static TestOperationResultStorage Instance { get; } = new TestOperationResultStorage();
```

Comment:
You can initialize this Instance using Settings instead of calling InitAsync later.

Reply:
Changed this to be a static class, but still using InitAsync